package.json

```diff
@@ -16,12 +17,15 @@
     "@babel/runtime": "^7.6.0",
     "@hapi/hapi": "^18.4.0",
     "async": "^3.1.0",
-    "mysql": "^2.17.1"
+    "config": "^3.2.2",
+    "mysql": "^2.17.1",
+    "open": "^6.4.0"
```

Comment:
`open` is a small library used to `OPEN` we browser after we start server.

Documentation: https://www.npmjs.com/package/open